src/pkg/caendr/caendr/models/error.py

```diff
@@ -109,6 +109,13 @@ def __init__(self, msg, line: int=None):
 class GoogleSheetsParseError(InternalError):
   description = "Unable to parse Google Sheets document"
 
+class ExternalMarkdownRenderError(InternalError):
+  def __init__(self, url, src):
+    self.url = url
+    self.src = src
+    self.description = f'Unable to render markdown file from {url}: {src}'
+    super().__init__()

```

Comment:
consider moving `super().__init__()` to the top of the method to ensure that any following assignments won't be [accidentally] overriden by a parent constructor.